recipe/meta.yaml

```diff
@@ -43,25 +29,25 @@ requirements:
     - gcc
     - python
```

Comment:
Could we split these so that `python` and below go into the new `host` section and `gcc` stays in build? `conda-build` 3 differentiates build tools from the build environment, which would be good to take advantage of here.

Reply:
Have pushed a fix to your PR to fix this. Hope that is ok.